### PR TITLE
[docs] Doc for a raft flag: --evict_failed_follower

### LINKED ISSUE #15207: [Docs] Document --evict_failed_followers flag
### Description

Refer to  https://github.com/yugabyte/yugabyte-db/issues/15003 for details

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #15207 
Build preview: https://deploy-preview-15258--infallible-bardeen-164bc9.netlify.app/preview/reference/configuration/yb-master/#evict-failed-followers
https://deploy-preview-15258--infallible-bardeen-164bc9.netlify.app/preview/reference/configuration/yb-tserver/#evict-failed-followers

### CODE REVIEW

File: docs/content/preview/reference/configuration/yb-master.md [hunk 1]
```diff
@@ -252,6 +252,14 @@ Default: `7200` (2 hours)
 
 The `--follower_unavailable_considered_failed_sec` value should match the value for [`--log_min_seconds_to_retain`](#log-min-seconds-to-retain).
 
+##### --evict_failed_followers
+
+Failed followers will be evicted from the Raft group and the data will be re-replicated.
+
+Default: `true`
+
+Note that it is not recommended to set the flag to true for masters as you cannot automatically recover a failed master once its evicted.
```
Comment (ionthegeek) [inline docs/content/preview/reference/configuration/yb-master.md:261]: Typo here - "its" should be "it's" or "it has been"

File: docs/content/preview/reference/configuration/yb-master.md [hunk 2]
```diff
@@ -252,6 +252,14 @@ Default: `7200` (2 hours)
 
 The `--follower_unavailable_considered_failed_sec` value should match the value for [`--log_min_seconds_to_retain`](#log-min-seconds-to-retain).
 
+##### --evict_failed_followers
+
+Failed followers will be evicted from the Raft group and the data will be re-replicated.
+
+Default: `true`
```
Comment (ionthegeek) [inline docs/content/preview/reference/configuration/yb-master.md:259]: The default for masters is `false`.

**Review by ionthegeek — CHANGES_REQUESTED** [review]
A couple of small but important changes needed in the master doc.

**Review by ionthegeek — APPROVED** [review]
Looks good!

**Review by ddhodge — APPROVED** [review]
LGTM

**Pull request merged**